Tools-Override/tests.targets

```diff
@@ -55,11 +60,72 @@
+      <DotNetCliSharedFrameworkSdkRuntimeConfig Include="$(TestDotNetCliPath)\sdk\**\dotnet.runtimeconfig.json" />
+      <LiveDotNetCliComponents Include="$(RuntimePath)\*.*" Exclude="$(RuntimePath)\xunit.console.netcore.exe" />
+    </ItemGroup>
+  
+    <Copy SourceFiles="@(DotNetCliFiles)" 
+          DestinationFiles="@(DotNetCliFiles -> '$(TestDotNetCliPath)\%(RecursiveDir)%(Filename)%(Extension)')" />
+    <Copy SourceFiles="@(DotNetCliSharedFrameworkHost)" 
+          DestinationFiles="@(DotNetCliSharedFrameworkHost -> '$(TestDotNetCliPath)\host\%(RecursiveDir)%(Filename)%(Extension)')" />
+    <Copy SourceFiles="@(DotNetCliSharedFrameworkSdk)" 
+          DestinationFiles="@(DotNetCliSharedFrameworkSdk -> '$(TestDotNetCliPath)\sdk\%(RecursiveDir)%(Filename)%(Extension)')" />
+    <ItemGroup>     
+      <DotNetCliSharedFrameworkSdkRuntimeConfig Include="$(TestDotNetCliPath)\sdk\**\dotnet.runtimeconfig.json" />
+    </ItemGroup>
+    <ReadLinesFromFile File="@(DotNetCliSharedFrameworkSdkRuntimeConfig)" >  
+        <Output TaskParameter="Lines"  
+                ItemName="LinesFromDotNetCliSharedFrameworkSdkRuntimeConfig"/> 
+    </ReadLinesFromFile>
+    <ItemGroup>
+       <LinesToWriteDotNetCliSharedFrameworkSdkRuntimeConfig Include="$([System.String]::Copy('%(LinesFromDotNetCliSharedFrameworkSdkRuntimeConfig.Identity)').Replace('1.0.0','9.9.9'))" />
+       <LinesToWriteDotNetCliSharedFrameworkSdkRuntimeConfig Include="};}" />
+    </ItemGroup>
+    <WriteLinesToFile  
+            File="@(DotNetCliSharedFrameworkSdkRuntimeConfig)"  
+            Lines="@(LinesToWriteDotNetCliSharedFrameworkSdkRuntimeConfig)"  
+            Overwrite="true" />
+    <Copy SourceFiles="@(DotNetCliSharedFrameworkSdkRuntimeConfig)" 
+          DestinationFiles="@(DotNetCliSharedFrameworkSdkRuntimeConfig -> '$(ObjDir)\xunit\xunit.console.netcore.runtimeconfig.json')" />
+    <Copy SourceFiles="@(DotNetCliSharedFramework)" 
+          DestinationFiles="@(DotNetCliSharedFramework -> '$(TestDotNetCliPath)\shared\Microsoft.NETCore.App\9.9.9\%(Filename)%(Extension)')" />
+    <Copy SourceFiles="@(LiveDotNetCliComponents)" 
+          DestinationFiles="@(LiveDotNetCliComponents -> '$(TestDotNetCliPath)\shared\Microsoft.NETCore.App\9.9.9\%(Filename)%(Extension)')" />
+  
+    <ItemGroup>
+      <OriginalDepsJsonPathItemGroup Include="$(TestDotNetCliPath)\shared\Microsoft.NETCore.App\9.9.9\*.deps.json" />
+    </ItemGroup>
+    <PropertyGroup>
+      <OriginalDepsJsonPath>@(OriginalDepsJsonPathItemGroup)</OriginalDepsJsonPath>
+    </PropertyGroup>
+    <GenerateDepsJson DepsJsonPath="$(OriginalDepsJsonPath)" />
+  </Target>
 
   <!-- The Code Coverage targets will override TestHost and TestCommandLine if coverage is enabled -->
   <Import Project="$(MSBuildThisFileDirectory)CodeCoverage.targets" />
+  
+  <Target Name="AddRuntimeConfig" >
+    <ItemGroup>
+       <SupplementalTestData Include="$(ObjDir)\xunit\xunit.console.netcore.runtimeconfig.json" />
```

Comment:
I tried converting this to Content but copying didn't seem to be working. The target that is supposed to do so is _CopyFilesMarkedCopyLocal and while it did appear in the pp log it didn't seem to be performing a copy.